[2022-06-03 22:04:16,500] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: UK_COVID19_dag.Extract_UK_Data 2022-06-03T21:43:20.220927+00:00 [queued]>
[2022-06-03 22:04:16,808] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: UK_COVID19_dag.Extract_UK_Data 2022-06-03T21:43:20.220927+00:00 [queued]>
[2022-06-03 22:04:16,811] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 22:04:16,856] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-06-03 22:04:16,891] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 22:04:17,244] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Extract_UK_Data> on 2022-06-03T21:43:20.220927+00:00
[2022-06-03 22:04:17,811] {standard_task_runner.py:52} INFO - Started process 100 to run task
[2022-06-03 22:04:18,158] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'UK_COVID19_dag', 'Extract_UK_Data', '2022-06-03T21:43:20.220927+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag-checkpoint.py', '--cfg-path', '/tmp/tmp1vu75are', '--error-file', '/tmp/tmpaazv3aq7']
[2022-06-03 22:04:18,193] {standard_task_runner.py:77} INFO - Job 9: Subtask Extract_UK_Data
[2022-06-03 22:04:19,319] {logging_mixin.py:104} INFO - Running <TaskInstance: UK_COVID19_dag.Extract_UK_Data 2022-06-03T21:43:20.220927+00:00 [running]> on host db1a6c3b41a0
[2022-06-03 22:04:20,023] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Huda
AIRFLOW_CTX_DAG_ID=UK_COVID19_dag
AIRFLOW_CTX_TASK_ID=Extract_UK_Data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T21:43:20.220927+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T21:43:20.220927+00:00
[2022-06-03 22:08:01,151] {python.py:118} INFO - Done. Returned value was: None
[2022-06-03 22:08:01,287] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=UK_COVID19_dag, task_id=Extract_UK_Data, execution_date=20220603T214320, start_date=20220603T220416, end_date=20220603T220801
[2022-06-03 22:08:01,671] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-03 22:08:01,992] {local_task_job.py:146} INFO - Task exited with return code 0
